umpleonline/scripts/umple_page.js

```diff
@@ -1092,6 +1108,7 @@ Page.getSelectedExample = function()
   if(theExampleType == "cdModels") {
     var requiresGvClass = false; // Some class diagrams  are too complex to edit
     inputExample = jQuery("#inputExample option:selected").val();
+    sessionStorage.setItem("savedSelection", inputExample);
```

Comment:
Sessionstorage was indeed residual code and I made sure to remove it. I also modified the logic on which setexamples is operating so it might hopefully work on Safari now: Rather than searching for the appropriate value in all of the 4 menus, this search will stop once the requested example is found. This means that if the requested example is found in inputExample2, it will first go through inputexample and then to the former before returning nothing and thus not going through inputexample 3 and 4.